src/google-maps/deprecated-map-marker-clusterer/deprecated-map-marker-clusterer.spec.ts

```diff
@@ -0,0 +1,375 @@
+import {Component, ViewChild} from '@angular/core';
```

Comment:
I'm hesitant to add new errors to the existing component since we're already deprecating it anyway. Currently it's a no-op if somebody passes in advanced markers and it'll continue working like that after this change.

Reply:
Makes sense!